### PR TITLE
Document two-level lists

### PR DESCRIPTION
**Pull request by abarth**
These are the last dartdocs needed in the material library.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @HansMuller

File: packages/flutter/lib/src/material/two_level_list.dart [hunk 1]
```diff
@@ -16,22 +16,61 @@ import 'theme_data.dart';
 
 const Duration _kExpand = const Duration(milliseconds: 200);
 
+/// An item in a [TwoLevelList] or a [TwoLevelSublist]
```
Comment (dragostis) [inline packages/flutter/lib/src/material/two_level_list.dart]: Missing period.

File: packages/flutter/lib/src/material/two_level_list.dart [hunk 2]
```diff
@@ -16,22 +16,61 @@ import 'theme_data.dart';
...
   final Widget title;
+
+  /// A widget to display after the title.
+  ///
+  /// Typically an [Icon] widget.
   final Widget trailing;
+
+  /// Whether this list item is interactive.
+  ///
+  /// If false, this list item is styled with the disabled color from the
```
Comment (dragostis) [inline packages/flutter/lib/src/material/two_level_list.dart]: 'false' should be between `s.

**Review by dragostis — APPROVED** [review]
LGTM.

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/material/two_level_list.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/material/two_level_list.dart]: Done

File: packages/flutter/lib/src/material/two_level_list.dart (hunk 2 above)
Comment (abarth) [inline packages/flutter/lib/src/material/two_level_list.dart]: Done.

**Pull request merged**